### ISSUE TITLE
Failure with dev version of testthat

### ISSUE THREAD

**Issue by hadley** [gavinsimpson/permute#3]
```
checking tests ... ERROR
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  
  > ## Test `permute` using the `testthat` package
  > 
  > ## Setup
  > library(testthat)
  > library(permute)
  > 
  > ## Runs the tests in inst/tests
  > test_package("permute")
  Error in eval(expr, envir, enclos) : 
    could not find function "library_if_available"
  Calls: test_package ... test_dir -> lapply -> FUN -> sys.source2 -> eval -> eval
  Execution halted
```

**Comment by gavinsimpson** [comment]
Thanks @hadley I'll fix that and be ready to ship a new version to CRAN.

**Comment by gavinsimpson** [comment]
Fixed in 0a53b6a76152b6360c11de069c823d150d1bb5c2
